db/seeds.rb

```diff
@@ -6,6 +6,13 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 if Rails.env.development?
+  AdminUser.delete_all
+  User.delete_all
+  UsersProject.delete_all
+  Metric.delete_all
+  Technology.delete_all
+  Project.delete_all
```

Comment:
Why are we doing this?

Reply:
to keep the consistency in dev-database, doing a delete from all the models avoid duplication/validation errors from the models previously created, every time we run a seed it just generate the current data and doesn't get mixed with previous data

Reply:
Wouldn't be better to just make a `rake db:reset`?

Reply:
you are right, better a `reset`. going to remove it 👍